client/src/pages/contentPages/Groups.js

```diff
@@ -1,23 +1,278 @@
-import React from 'react';
-import { Grid, Typography, Container } from '@material-ui/core';
+import React, { useState, useEffect, useMemo, useCallback } from 'react';
+import { Grid, Typography, Container, Button, Modal, FormHelperText, MenuItem, Select, TextField } from '@material-ui/core';
+import {
+  baseStyle,
+  activeStyle,
+  acceptStyle,
+  rejectStyle,
+} from '../../components/Group/GroupPicture.styles';
+import { Tooltip, Box} from '@material-ui/core';
+import { useDropzone } from 'react-dropzone';
+import axios from 'axios';
+
 import { makeStyles } from '@material-ui/core/styles';
 import GroupCard from '../../components/Group/GroupCard';
-import axios from 'axios';
+import defaultImage from '../../images/upload_placeholder.png';
+import { useSnackbar } from 'notistack';
 import { useGlobalContext } from '../../context/studyappContext';
+import * as actions from '../../context/actions';
 
 const useStyles = makeStyles((theme) => ({
     cardGrid: {
       paddingTop: theme.spacing(8),
       paddingBottom: theme.spacing(8),
     },
+    button: {
+      color: '#FFF',
+      background: theme.palette.primary.gradient,
+      marginTop: "60px",
+      marginRight: "60px",
+      textTransform: 'none',
+      height: '3rem',
```

Comment:
If theme is updated properly from dev you won't need: height, color or background here if the button is color="primary"